lib/logstash/outputs/rabbitmq.rb

```diff
@@ -44,22 +46,24 @@ class RabbitMQ < LogStash::Outputs::Base
       def register
         connect!
         @hare_info.exchange = declare_exchange!(@hare_info.channel, @exchange, @exchange_type, @durable)
```

Comment:
Is this still required?

Reply:
Yes, we'll want to declare the exchange on register, otherwise if this fails, it will be surprising and in the middle of execution.